README.md

```diff
@@ -164,6 +140,8 @@ A _resource_ is a **category**, **product**, **customer** or **order**. Any time
 
 All calls to these endpoints have to be done from your backend - therefore, we don't provide any specific code examples as the implementation is bound to your backend logic and programming language.
 
+:warning: Each backend call except `/customer` **must** be sent with additional custom header called `X-Digest`, which is a HMAC-SHA256 digest generated by using the request body string as the message and your Metrilo API secret as the key. [This list](https://github.com/danharper/hmac-examples) provides great examples in different programming languages, which can help you with the implementation. 
```

Comment:
Use :information_source: `:information_source:`. Don't scare users with warnings. The worst thing that can happen is that they'll receive an authentication error and if they can't figure it out, we have support to contact.